apiserver/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ray-project/kuberay/apiserver
 
-go 1.17
+go 1.19
```

Comment:
@kevin85421 may need to update the go version in the table https://github.com/ray-project/kuberay/blob/master/apiserver/DEVELOPMENT.md#kuberay-api-server-user-guide